frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick-start-access-code-snippet.feature

```diff
@@ -5,50 +5,49 @@ Feature: Access a code snippet from a quick start
 
         Background:
             Given user is at developer perspective
-              And Get started with spring CR Quick Start is available
+              And user has applied copy-execute-demo.yaml
 
 
         @regression @to-do
         Scenario: Copy to clipboard tooltip with code snippet: QS-02-TC01
-            Given user is in Quick Starts catalog page
-             When user clicks on "Get started with spring" Quick Starts card
+            Given user is at Quick Starts catalog page
+             When user clicks on "Create ruby app" Quick Starts card
```

Comment:
you need a scenario to create  `create ruby app`. It's not included beforehand.

Reply:
I've included in background statement with Yaml present in testData